straxen/plugins/led_cal/led_calibration.py

```diff
@@ -55,21 +56,23 @@ class LEDCalibration(strax.Plugin):
         help="Window (samples) for baseline calculation.",
     )
 
-    default_led_window = straxen.URLConfig(
-        default=(78, 107),
+    minimum_led_position = straxen.URLConfig(
```

Comment:
Is there a reference for this change? It seems that we change the double-sided range to a single lower limit for the triggering time. Isn't the original structure more robust?

Reply:
Thanks for the comment, this is actually the major change: In the previous version, I used a fixed default window, given by the old setting. However that means we see a different amount of underamplified PE, which we do not trigger on, depending on where they happen to be relative to the window. Now there is no default window anymore, instead if there is no hit found in a record, I use the mode of the distribution of all hits in one chunk.
Additionally the previous was redundant. If you wanted to change the window length, you had to remember to change both the `led_hit_extension` and the `default_led_window`. Now, `led_hit_extension` is always used.

Reply:
Thanks @tflehmke. It's clear to me now